front/app/containers/Admin/messaging/AutomatedEmails/CampaignRow.tsx

```diff
@@ -59,10 +56,32 @@ const CampaignRow = ({
     }
   };
 
-  const isEditingEnabled = useFeatureFlag({
-    name: 'customised_automated_emails',
-  });
+  const isEditingEnabled =
```

Comment:
Not fully following the business logic here. In which cases is there no context, and why is in those cases editing allowed? Or is it allowed if `campaign.attributes.enabled`? Maybe good to clarify a bit with a comment?